lighthouse-core/scripts/legacy-javascript/summary-sizes.txt

```diff
@@ -1,170 +1,170 @@
-  11236        es6-array-map/main.bundle.min.js
-  11231        es6-object-keys/main.bundle.min.js
-  11203        es6-reflect-own-keys/main.bundle.min.js
-  11190        es6-object-define-properties/main.bundle.min.js
-  10884        es6-reflect-set/main.bundle.min.js
-  10406        es6-reflect-get/main.bundle.min.js
-  10367        es7-array-includes/main.bundle.min.js
-  10018        es6-string-ends-with/main.bundle.min.js
-   9972        es6-string-starts-with/main.bundle.min.js
-   9884        es6-reflect-set-prototype-of/main.bundle.min.js
-   9770        es6-array-fill/main.bundle.min.js
-   9750        es6-array-reduce-right/main.bundle.min.js
-   9726        es6-object-set-prototype-of/main.bundle.min.js
-   9724        es6-array-reduce/main.bundle.min.js
-   9605        es7-string-pad-start/main.bundle.min.js
-   9598        es7-string-pad-end/main.bundle.min.js
-   9576        es6-object-get-own-property-descriptor/main.bundle.min.js
-   9502        es6-array-last-index-of/main.bundle.min.js
-   9359        es6-string-includes/main.bundle.min.js
-   9240        es6-reflect-delete-property/main.bundle.min.js
-   9217        es6-reflect-get-own-property-descriptor/main.bundle.min.js
-   9164        es6-object-prevent-extensions/main.bundle.min.js
-   9137        es6-number-parse-int/main.bundle.min.js
-   9131        es6-number-parse-float/main.bundle.min.js
-   9085        es6-object-freeze/main.bundle.min.js
-   9071        es6-object-seal/main.bundle.min.js
-   9053        es6-object-get-prototype-of/main.bundle.min.js
-   8887        es6-string-raw/main.bundle.min.js
-   8711        es6-reflect-get-prototype-of/main.bundle.min.js
-   8637        es6-string-trim/main.bundle.min.js
-   8489        es6-string-from-code-point/main.bundle.min.js
-   8467        es6-date-to-iso-string/main.bundle.min.js
-   8457        es6-string-code-point-at/main.bundle.min.js
-   8261        es6-string-repeat/main.bundle.min.js
-   8237        es6-array-of/main.bundle.min.js
-   8235        es6-date-to-json/main.bundle.min.js
-   8048        es6-reflect-define-property/main.bundle.min.js
-   8042        es6-object-is-extensible/main.bundle.min.js
-   8014        es6-object-is-sealed/main.bundle.min.js
-   8014        es6-object-is-frozen/main.bundle.min.js
-   7986        es6-reflect-apply/main.bundle.min.js
-   7920        es6-number-is-safe-integer/main.bundle.min.js
-   7858        es6-array-is-array/main.bundle.min.js
-   7807        es6-reflect-prevent-extensions/main.bundle.min.js
-   7787        es6-number-is-integer/main.bundle.min.js
-   7737        es6-reflect-is-extensible/main.bundle.min.js
-   7652        es6-object-define-property/main.bundle.min.js
-   7583        es6-reflect-has/main.bundle.min.js
-   7550        es6-date-now/main.bundle.min.js
-   6037        es6-date-to-string/main.bundle.min.js
-   3454        es6-function-name/main.bundle.min.js
+  47832        es6-typed-uint8-clamped-array/main.bundle.min.js
```

Comment:
sizes changed because modules keys in the bundle are now numbers instead of strings.